encode/hevcencode.c

```diff
@@ -1844,6 +1807,7 @@ static int process_cmdline(int argc, char *argv[])
             frame_rate = atoi(optarg);
             break;
         case 'o':
+            free(coded_fn);
             coded_fn = strdup(optarg);
```

Comment:
mem leak? where mem allocated? and where leaked?

Reply:
Issue: Overwriting coded_fn in coded_fn = strdup(optarg) leaks the storage that coded_fn points to.
So need to free the pre-storage that coded_fn points to.